interface/billing/billing_report.php

```diff
@@ -65,59 +49,60 @@
         $results = $db->Execute($sql);
         $billings = array();
         if ($results->RecordCount() == 0) {
-            echo xlt("No Bills Found to Include in OFX Export")."<br>";
+            echo xlt("No Bills Found to Include in OFX Export") . "<br>";
         } else {
-            while (!$results->EOF) {
+            while (! $results->EOF) {
                 $billings[] = $results->fields;
                 $results->MoveNext();
             }
-
             $ofx = new OFX($billings);
             header("Pragma: public");
             header("Expires: 0");
             header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
             header("Content-Disposition: attachment; filename=openemr_ofx.ofx");
             header("Content-Type: text/xml");
             echo $ofx->get_OFX();
-            exit;
+            exit();
         }
     }
 }
 
-//global variables:
-$from_date     = isset($_POST['from_date'])  ? $_POST['from_date']  : date('Y-m-d');
-$to_date       = isset($_POST['to_date'  ])  ? $_POST['to_date'  ]  : '';
-$code_type     = isset($_POST['code_type'])  ? $_POST['code_type']  : 'all';
-$unbilled      = isset($_POST['unbilled' ])  ? $_POST['unbilled' ]  : 'on';
+// global variables:
+$from_date = isset($_POST['from_date']) ? $_POST['from_date'] : date('Y-m-d');
+$to_date = isset($_POST['to_date']) ? $_POST['to_date'] : '';
+$code_type = isset($_POST['code_type']) ? $_POST['code_type'] : 'all';
+$unbilled = isset($_POST['unbilled']) ? $_POST['unbilled'] : 'on';
 $my_authorized = isset($_POST["authorized"]) ? $_POST["authorized"] : '';
 
-
 // This tells us if only encounters that appear to be missing a "25" modifier
 // are to be reported.
-$missing_mods_only = (isset($_POST['missing_mods_only']) && !empty($_POST['missing_mods_only']));
+$missing_mods_only = (isset($_POST['missing_mods_only']) && ! empty($_POST['missing_mods_only']));
 
 $left_margin = isset($_POST["left_margin"]) ? $_POST["left_margin"] : $GLOBALS['cms_left_margin_default'];
-$top_margin  = isset($_POST["top_margin"]) ? $_POST["top_margin" ] : $GLOBALS['cms_top_margin_default'];
-
-$ofrom_date  = $from_date;
-$oto_date    = $to_date;
-$ocode_type  = $code_type;
-$ounbilled   = $unbilled;
+$top_margin = isset($_POST["top_margin"]) ? $_POST["top_margin"] : $GLOBALS['cms_top_margin_default'];
+if ($ub04_support) {
+    $left_ubmargin = isset($_POST["left_ubmargin"]) ? $_POST["left_ubmargin"] : $GLOBALS['left_ubmargin_default'];
+    $top_ubmargin = isset($_POST["top_ubmargin"]) ? $_POST["top_ubmargin"] : $GLOBALS['top_ubmargin_default'];
+}
+$ofrom_date = $from_date;
+$oto_date = $to_date;
+$ocode_type = $code_type;
+$ounbilled = $unbilled;
 $oauthorized = $my_authorized;
 ?>
-
+<!DOCTYPE html >
 <html>
 <head>
 <?php if (function_exists('html_header_show')) {
     html_header_show();
-} ?>
+}?>
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-11-3/index.js"></script>
```

Comment:
actually could also remove the common.js include at line 349 and then change the Header call to:
``````
Header::setupHeader(['datetime-picker','common']);
``````